### PR TITLE
alphabetically reorder acknowledgements list and make it more thorough.

### PR DESCRIPTION
**Pull request by Sakurann**
For the awareness purposes, adding for review those who I added in the list. will merge when few approvals come in.

cc some of those whom I could not add as reviewers @TakahikoKawasaki, @cobward

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

File: openid-4-verifiable-presentations-1_0.md [hunk 1]
```diff
@@ -1661,7 +1661,7 @@ This specification registers the following JWS header name in the IANA "JSON Web
 
 # Acknowledgements {#Acknowledgements}
 
-We would like to thank John Bradley, Brian Campbell, David Chadwick, Giuseppe De Marco, Daniel Fett, George Fletcher, Fabian Hauck, Joseph Heenan, Alen Horvat, Andrew Hughes, Edmund Jay, Michael B. Jones, Gaurav Khot, Ronald Koenig, Mark Haine, Adam Lemmon, Kenichi Nakamura, Nat Sakimura, Arjen van Veen, and Jacob Ward for their valuable feedback and contributions that helped to evolve this specification.
+We would like to thank Adam Lemmon, Alen Horvat, Andrew Hughes, Andrii Deinega, Arjen van Veen, Brian Campbell, Christian Bormann, David Chadwick, Daniel Fett, Daniel McGrogan, David Waite, Edmund Jay, Fabian Hauck, Gabe Cohen, Gaurav Khot, George Fletcher, Giuseppe De Marco, Jacob Ideskog, Jacob Ward, Jeremie Miller, John Bradley, Joseph Heenan, Judith Kahrer, Kenichi Nakamura, Mark Dobrinic, Mark Haine, Markus Kreusch, Michael B. Jones, Nat Sakimura, Oliver Terbu, Paul Bastian, Pedro Felix, Richard Barnes, Roland Hedberg, Rolson Quadras, Ronald Koenig, Takahiko Kawasaki, Timo Glasta, Tom Jones, Vittorio Bertocci for their valuable feedback and contributions to this specification.
```
Comment (Sakurann) [inline openid-4-verifiable-presentations-1_0.md:1664]:
```suggestion
We would like to thank Adam Lemmon, Alen Horvat, Andrew Hughes, Andrii Deinega, Arjen van Veen, Brian Campbell, Christian Bormann, David Chadwick, Daniel Fett, Daniel McGrogan, David Waite, Edmund Jay, Fabian Hauck, Gabe Cohen, Gaurav Khot, George Fletcher, Giuseppe De Marco, Jacob Ideskog, Jacob Ward, Jeremie Miller, John Bradley, Joseph Heenan, Judith Kahrer, Kenichi Nakamura, Mark Dobrinic, Mark Haine, Markus Kreusch, Michael B. Jones, Nat Sakimura, Niels Klomp, Oliver Terbu, Paul Bastian, Pedro Felix, Richard Barnes, Roland Hedberg, Rolson Quadras, Ronald Koenig, Takahiko Kawasaki, Timo Glasta, Tom Jones, Vittorio Bertocci for their valuable feedback and contributions to this specification.
```

**Review by tplooker — APPROVED** [review]

**Review by paulbastian — APPROVED** [review]

**Review by TimoGlastra — APPROVED** [review]

**Review by c2bo — APPROVED** [review]

File: openid-4-verifiable-presentations-1_0.md [hunk 2]
```diff
@@ -1661,7 +1661,7 @@ This specification registers the following JWS header name in the IANA "JSON Web
 
 # Acknowledgements {#Acknowledgements}
 
-We would like to thank John Bradley, Brian Campbell, David Chadwick, Giuseppe De Marco, Daniel Fett, George Fletcher, Fabian Hauck, Joseph Heenan, Alen Horvat, Andrew Hughes, Edmund Jay, Michael B. Jones, Gaurav Khot, Ronald Koenig, Mark Haine, Adam Lemmon, Kenichi Nakamura, Nat Sakimura, Arjen van Veen, and Jacob Ward for their valuable feedback and contributions that helped to evolve this specification.
+We would like to thank Adam Lemmon, Alen Horvat, Andrew Hughes, Andrii Deinega, Arjen van Veen, Brian Campbell, Christian Bormann, David Chadwick, Daniel Fett, Daniel McGrogan, David Waite, Edmund Jay, Fabian Hauck, Gabe Cohen, Gaurav Khot, George Fletcher, Giuseppe De Marco, Jacob Ideskog, Jacob Ward, Jeremie Miller, John Bradley, Joseph Heenan, Judith Kahrer, Kenichi Nakamura, Mark Dobrinic, Mark Haine, Markus Kreusch, Michael B. Jones, Nat Sakimura, Niels Klomp, Oliver Terbu, Paul Bastian, Pedro Felix, Richard Barnes, Roland Hedberg, Rolson Quadras, Ronald Koenig, Takahiko Kawasaki, Timo Glasta, Tom Jones, Vittorio Bertocci for their valuable feedback and contributions to this specification.
```
Comment (Sakurann) [inline openid-4-verifiable-presentations-1_0.md:1664]:
```suggestion
We would like to thank Adam Lemmon, Alen Horvat, Andrew Hughes, Andrii Deinega, Arjen van Veen, Brian Campbell, Christian Bormann, David Chadwick, Daniel Fett, Daniel McGrogan, David Waite, Edmund Jay, Fabian Hauck, Gabe Cohen, Gaurav Khot, George Fletcher, Giuseppe De Marco, Jacob Ideskog, Jacob Ward, Jeremie Miller, John Bradley, Joseph Heenan, Judith Kahrer, Kenichi Nakamura, Mark Dobrinic, Mark Haine, Markus Kreusch, Michael B. Jones, Nat Sakimura, Niels Klomp, Paul Bastian, Pedro Felix, Richard Barnes, Roland Hedberg, Rolson Quadras, Ronald Koenig, Takahiko Kawasaki, Timo Glasta, Tom Jones, Vittorio Bertocci for their valuable feedback and contributions to this specification.
```

**Review by jogu — APPROVED** [review]

**Review by selfissued — APPROVED** [review]

**Review by selfissued — COMMENTED** [review]
I'll point out that the names were already in alphabetical order *by last name*, which is the typical sorting.

I'm not going to stand in the way of this on that basis because I think it's important to acknowledge everyone in the list, but I do find the change in sorting convention odd.  Why was this changed?

File: openid-4-verifiable-presentations-1_0.md [hunk 3]
```diff
@@ -1661,7 +1661,7 @@ This specification registers the following JWS header name in the IANA "JSON Web
 
 # Acknowledgements {#Acknowledgements}
 
-We would like to thank John Bradley, Brian Campbell, David Chadwick, Giuseppe De Marco, Daniel Fett, George Fletcher, Fabian Hauck, Joseph Heenan, Alen Horvat, Andrew Hughes, Edmund Jay, Michael B. Jones, Gaurav Khot, Ronald Koenig, Mark Haine, Adam Lemmon, Kenichi Nakamura, Nat Sakimura, Arjen van Veen, and Jacob Ward for their valuable feedback and contributions that helped to evolve this specification.
+We would like to thank Adam Lemmon, Alen Horvat, Andrew Hughes, Andrii Deinega, Arjen van Veen, Brian Campbell, Christian Bormann, David Chadwick, Daniel Fett, Daniel McGrogan, David Waite, Edmund Jay, Fabian Hauck, Gabe Cohen, Gaurav Khot, George Fletcher, Giuseppe De Marco, Jacob Ideskog, Jacob Ward, Jeremie Miller, John Bradley, Joseph Heenan, Judith Kahrer, Kenichi Nakamura, Mark Dobrinic, Mark Haine, Markus Kreusch, Michael B. Jones, Nat Sakimura, Niels Klomp, Paul Bastian, Pedro Felix, Richard Barnes, Roland Hedberg, Rolson Quadras, Ronald Koenig, Takahiko Kawasaki, Timo Glasta, Tom Jones, Vittorio Bertocci for their valuable feedback and contributions to this specification.
```
Comment (Sakurann) [inline openid-4-verifiable-presentations-1_0.md:1664]:
```suggestion
We would like to thank Richard Barnes, Paul Bastian, Vittorio Bertocci, Christian Bormann, John Bradley, Brian Campbell, Gabe Cohen, David Chadwick, Andrii Deinega, Giuseppe De Marco, Mark Dobrinic, Daniel Fett, Pedro Felix, George Fletcher, Timo Glasta, Mark Haine, Fabian Hauck, Roland Hedberg, Joseph Heenan, Alen Horvat, Andrew Hughes, Jacob Ideskog, Edmund Jay, Michael B. Jones, Tom Jones, Judith Kahrer, Takahiko Kawasaki, Gaurav Khot, Niels Klomp, Ronald Koenig, Markus Kreusch, Adam Lemmon, Daniel McGrogan, Jeremie Miller, Kenichi Nakamura, Rolson Quadras, Nat Sakimura, Arjen van Veen, David Waite, Jacob Ward for their valuable feedback and contributions to this specification.
```

Comment (Sakurann) [reply openid-4-verifiable-presentations-1_0.md:1664]: reordering alphabetically based on the last name

**Comment by Sakurann** [comment]
VCI spec was not ordered at all and I automatically reordered both based on first name without realizing VP spec was already ordered. i am ok ordering based on the last name

**Comment by selfissued** [comment]
Looks good now!

**Pull request merged**